packages/workspace/src/core/file-utils.ts

```diff
@@ -171,7 +171,7 @@ export function readWorkspaceConfig(opts: {
   format: 'angularCli' | 'nx';
   path?: string;
 }) {
-  const ws = new Workspaces(opts.path);
+  const ws = new Workspaces(opts.path || '');
```

Comment:
Hm it should probably be `process.cwd()` and not `''` though I guess both are the same?